standalone/docker-entrypoint.sh

```diff
@@ -16,6 +16,12 @@ if [ ! -f "$AUTO_START" ]; then
     cp "$DEFAULT_AUTO_START" "$AUTO_START"
 fi
 
+# remove leftover lockfiles from possible unclean shutdowns before startup
+if [ "${REMOVE_LOCK_FILES}" ] && [ "${REMOVE_LOCK_FILES}" != false ]; then
```

Comment:
A couple lines above we only interpret a flag as set if it is explicitly `true`. Feels like the safer approach somehow or at least consistent with what we have (whitelist vs. blacklist).